gobcore/sources/relations.md

```diff
@@ -107,18 +109,155 @@ To get the complete data of an entity it has to be joined with its relations.
 Relations become like regular entities within GOB. They are updated using events and can be tracked over time. The regular entity attributes are part of each relation.
 
     gobid             GOB unique internal id
-    _id               Functional id, can be equal to technical id
+    _id               catalog.collection.id.attribute, eg gebieden.wijken.036XXX.ligt_in_stadsdeel
     _version          0.1
     _date_created
     _date_confirmed
     _date_modified
     _date_deleted
-    _source           functional source for the bronwaarde, e.g. 'AMSBI'
-    _application      technical source for the bronwaarde, e.g. 'Neuron'
-    _source_id        technical id for the bronwaarde
+    _source           
+    _application      
+    _source_id        _id.volgnummer        
     _last_event       last event that has "touched" the relation
     _hash             hash of the relation to allow for fast compares
 
 Relations can be a regular part of GOB by defining its structure in GOB Model. It's catalog can be set to gob and the collection to relations. The database table name becomes gob_relations.
 
 GOB Relations can be retrieved via the API like any other GOB entity.
+
+## Example
+
+BAG-Adressen:
+
+    {
+        _id: 1
+        ligt_in_wijk: {
+            bronwaarde: 'W1'
+        },
+        volgnummer: 1,
+        begin_geldigheid: 1-1-2001,
+        eind_geldigheid: 1-1-2007
+    }
+	
+    {
+        _id: 1
+        ligt_in_wijk: {
+            bronwaarde: 'W3'
+        },
+        volgnummer: 2,
+        begin_geldigheid: 1-1-2007,
+        eind_geldigheid: NULL
+    }
+
+Gebieden-Wijken:
+
+    {
+        _id: 1,
+        code: 'W1',
+        volgnummer: 1,
+        begin_geldigheid: 1-1-2000,
+        eind_geldigheid: 1-1-2003
+    }
+	
+    {
+        _id: 2,
+        code: 'W1',
+        volgnummer: 1,
+        begin_geldigheid: 1-1-2003,
+        eind_geldigheid: 1-1-2006
+    }
+	
+    {
+        _id: 1,
+        code: 'W1',
+        volgnummer: 2,
+        begin_geldigheid: 1-1-2006,
+        eind_geldigheid: NULL
+    }
+	
+    {
+        _id: 3,
+        code: 'W3',
+        volgnummer: 1,
+        begin_geldigheid: 1-1-2000,
+        eind_geldigheid: 1-1-2009
+    }
+	
+
+GOB-Relations:
+
+    {
+        _id: 'bag.adressen.ligt_in_wijk'
```

Comment:
The value of id is missing in this example. According to the specs above it should be:

catalog.collection.id.attribute, eg bag.adressen.1.ligt_in_wijk